lang_tests/run.rs

```diff
@@ -3,30 +3,42 @@ use std::{fs::read_to_string, path::PathBuf, process::Command};
 use lang_tester::LangTester;
 use lazy_static::lazy_static;
 use regex::{Regex, RegexBuilder};
-
-// FIXME: doesn't yet test the JIT.
-static BF_BINARIES: &[&str] = &["bf_base"];
+use std::collections::HashMap;
 
 lazy_static! {
     static ref EXPECTED: Regex = RegexBuilder::new("^\\[(.*?)^\\][ \t]*$")
         .multi_line(true)
         .dot_matches_new_line(true)
         .build()
         .unwrap();
+
+    // Maps binary name to a list of benchmarks to skip.
```

Comment:
Very cunning idea -- I like it!

Reply:
Did you see the bit where i replace the test with "ignore: ignore" if we want to skip it? I felt so dirty.

Reply:
Huh, no I didn't. Why don't you filter files in `test_file_filter` (it's explicitly meant for this sort of thing).

Reply:
I started by doing this, but realised that if you replace the test, you see that it was skipped (instead of appearing to be non-existent).

Reply:
Surely seeing that it's skipped is better than some super fragile file mutation thing?

Reply:
> We don't mutate any file. Just pass the test string as a "ignore: ignore".

Ah! OK, I'm fine with that.